core/templates/domain/entity_editor_browser_tabs_info/entity-editor-browser-tabs-info.model.spec.ts

```diff
@@ -30,8 +30,8 @@ describe('Entity Editor Browser Tabs Info', () => {
 
   beforeEach(() => {
     skillEditorBrowserTabsInfo = EntityEditorBrowserTabsInfo
-      .createFromObjectAndId(
-        skillEditorBrowserTabsInfoObject, 'skill_1');
+      .fromLocalStorageDict(
```

Comment:
If you're calling it a dict in one place, call it a dict in all place, otherwise it gets confusing (to have something like fromDict(object))

Reply:
Done. PTAL.